src/App/Pages/Accounts/TwoFactorPage.xaml.cs

```diff
@@ -145,6 +137,21 @@ private async void Methods_Clicked(object sender, EventArgs e)
             }
         }
 
+        private async void More_Clicked(object sender, EventArgs e)
+        {
+            if (DoOnce())
```

Comment:
This condition should be inverted. It's intended to prevent double-clicks, but at the moment it _requires_ a double-click.
```suggestion
            if (!DoOnce())
```